rules/visual-design-specifications/rule.md

```diff
@@ -1,6 +1,6 @@
 ---
 type: rule
-title: Do you determine specifications with a visual design?
+title: Do you conduct specification analysis by storyboarding?
 uri: visual-design-specifications
```

Comment:
1. Update the URI (make sure you update the folder name as per Wiki)

Reply:
We dont use a storyboard in specification review; we use low-fidelity and high-fidelity mockups.
Storyboarding is more closely associated with a prototype than a mockup.

Reply:
Agreed, forget about the specification review. This rule title and URI should talk about storyboards...

E.g. 

title: `Do you know when to use storyboards in software developement?`
uri: `storyboards`

Reply:
Changed to:

Do you use storyboards in software development?